.changeset/lovely-dogs-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@flatfile/plugin-autocast': patch
+---
+
+Adding keywords
```

Comment:
Since there are no code changes, does this need a changeset?

Reply:
it depends, if @ItsMeAra is using the package.json from npm, I think it might. If he's using it from github then no. Let us know @ItsMeAra